### PR TITLE
Expose WiFi.setLEDs(r, g, b) through AdafruitIO_AIRLIFT class.

### PR DESCRIPTION
**Pull request by ptbw**
The purpose of this change to to allow access to the setLEDs method in the WiFi class used by the Adafruit AirLift module. Making it very simple to add a status light to the class, and allow the visual inspection of the connection. 

The change adds a single public method to the AdafruitIO_AIRLIFT class which is only available if included using #define USE_AIRLIFT or similar, so will only be available to specifically including ..IO_AIRLIFT.h

I made a small modification to the AdafruitIO_LEDTest example (supplying a colour from Adafruit.IO):

```
void handleMessage(AdafruitIO_Data *data) {
  // print RGB values and hex value
  Serial.println("Received HEX: ");
  Serial.println(data->value());
   
  int red = data->toRed();
  int grn = data->toGreen();
  int blu = data->toBlue();
  Serial.println(red);
  Serial.println(grn);
  Serial.println(blu);
  io.setLEDs( red, grn, blu);
}
```
The example builds when using #define USE_AIRLIFT in the supplied config.h.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,4 +16,5 @@ platformio.ini
 # Our handy .gitignore for automation ease
 Doxyfile*
 doxygen_sqlite3.db
-html
\ No newline at end of file
+html
+latex
```
Comment (brentru) [inline .gitignore:20]: Please remove `latex` from the .gitignore

File: .travis.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+language: c
```
Comment (brentru) [inline .travis.yaml:1]: Please remove the `.travis.yaml` file, we use github actions on this repository

File: src/wifi/AdafruitIO_AIRLIFT.h [hunk 3]
```diff
@@ -93,6 +93,25 @@ class AdafruitIO_AIRLIFT : public AdafruitIO {
...
       delete _http;
   }
+  
+  
+  /**************************************************************************/
+  /*!
+  @brief  Allows setting of the AirLift RGB led from the 
+  Adafruit IO AirLift Class.
+  @param  r
+	  Red value
```
Comment (brentru) [inline src/wifi/AdafruitIO_AIRLIFT.h:103]: What are the limits of the `r` `g` `b` values? Is this value constrained (0->255?)

**Comment by brentru** [comment]
@ptbw  Thanks for making comments more explicit, I'll test on hardware and merge once it's passing the CI.

This PR is failing our CI build at the Clang step (https://github.com/adafruit/Adafruit_IO_Arduino/pull/133/checks?check_run_id=981225404#step:8:1).

**Comment by brentru** [comment]
> I made a small modification to the AdafruitIO_LEDTest example (supplying a colour from Adafruit.IO):

@ptbw  Do you want to add an example for this into `examples`? If not everything looks good and I can merge/release

**Comment by ptbw** [comment]
Go a head and merge, I can't really test any example on anything other than the Airlift feather wing.  But I have learnt a lot from this so thank you for your patience.

**Pull request merged**